src/main/java/model/Thing.java

```diff
@@ -0,0 +1,17 @@
+package model;
+
+public class Thing {
```

Comment:
We shouldn't have it and methods for it globally.

Reply:
What do you mean exactly? I made name private, but don't understand why the get and set methods shouldn't be global?